backend/test/integration/LobbyEvents.integration.test.js

```diff
@@ -25,17 +25,25 @@ describe('lobby events tests', () => {
         SocketIOServer.server.close();
     });
     /**
-     * This is a integration test for all lobby events.
-     * It encompases creating a game, Max players joining and reseting the game
+     * These are a integration tests for all lobby events.
```

Comment:
These are ~~a~~ integration tests for all lobby events.

Reply:
Good catch ill fit it up